frontend/src/js/external-forms/form-concept-group/FormConceptGroup.tsx

```diff
@@ -112,6 +112,17 @@ const FormConceptGroup = (props: Props) => {
   const newValue = props.newValue;
   const defaults = props.defaults || {};
 
+  // indicator if it should be scrolled down back to the dropZone
+  const [scrollToDropzone, setScrollToDropzone] = useState<boolean>(false);
+  const dropzoneRef = useRef<HTMLDivElement>(null);
+  useEffect(() => {
+    if (scrollToDropzone) {
+      dropzoneRef.current?.scrollIntoView({ behavior: "smooth" });
+      setScrollToDropzone(false);
+    }
+  });
```

Comment:
Why a useEffect without dependency array? If I understand correctly that will run the effect on every render. Then we don't even need the useEffect, right?

Reply:
Yes, that is correct, fixed that